dwa_local_planner/include/dwa_local_planner/dwa_planner.h

```diff
@@ -103,7 +104,7 @@ namespace dwa_local_planner {
        * @param drive_velocities The velocities to send to the robot base
        * @return The highest scoring trajectory. A cost >= 0 means the trajectory is legal to execute.
        */
-      base_local_planner::Trajectory findBestPath(
+      std::pair<base_local_planner::Trajectory, mbf_msgs::ExePathResult::_outcome_type> findBestPath(
```

Comment:
S: Change this so that the `Trajectory` is an output argument and return only the error code.

Reply:
I think the pair is cleaner;  
but maybe creating an alias for `mbf_msgs::ExePathResult::_outcome_type`, e.g. `ExePathOutcome` would make it more tidy